apps/docs/src/guide/wallets/connectors.md

```diff
@@ -4,6 +4,422 @@
+
+- `network` ([`Network`](../../api/Account/index.md#network))
+
+It will return a promise that will resolve to `true` if the network is successfully selected, otherwise `false`.
+
+It should throw an error if the network is not available or the network does _not_ exist.
+
+<<< @/../../../packages/account/src/connectors/fuel-connector.ts#fuel-connector-method-selectNetwork{ts:line-numbers}
+
+#### `addABI`
+
+The `addABI` method is used to add ABI information about a contract to the connector. This operation does no require an authorized connection.
+
+It requires two arguments:
+
+- `contractId` (`string`)
+- `abi` ([`FuelABI`](../../api/Account/index.md#fuelabi)).
+
+It will return a promise that will resolve to `true` if the ABI is successfully added, otherwise `false`.
+
+<<< @/../../../packages/account/src/connectors/fuel-connector.ts#fuel-connector-method-addABI{ts:line-numbers}
+
+#### `getABI`
+
+The `getABI` method is used to get the ABI information that is sorted about a contract.
+
+It requires a single argument:
+
+- `contractId` (`string`)
+
+It will return a promise that will resolve to the ABI information (as a [`FuelABI`](../../api/Account/index.md#fuelabi)) or `null` if the data is unavailable.
+
+<<< @/../../../packages/account/src/connectors/fuel-connector.ts#fuel-connector-method-getABI{ts:line-numbers}
+
+#### `hasABI`
+
+The `hasABI` method is used to check if the ABI information is available for a contract.
+
+It requires a single argument:
+
+- `contractId` (`string`)
+
+It will return a promise that will resolve to `true` if the ABI information is available, otherwise `false`.
+
+<<< @/../../../packages/account/src/connectors/fuel-connector.ts#fuel-connector-method-hasABI{ts:line-numbers}
+
+## Fuel SDK
+
+The `Fuel` manager provides the interface for interaction with the TS SDK and the wider Fuel ecosystem.
+
+It can be instantiated as follows:
+
+<<< @/../../docs-snippets/src/guide/wallets/connectors.test.ts#fuel-instantiation-1{ts:line-numbers}
+
+### Options
+
+There are a number of options that can be passed to the `Fuel` manager:
+
+- [`connectors`](#connectors)
+- [`storage`](#storage)
+- [`targetObject`](#targetObject)
+
+#### `connectors`
+
+The `connectors` option is used to provide a list of connectors that the `Fuel` manager can interact with. The manager will interact with the connectors that will in turn handle the communication with the respective wallet. You can find a list of all the connectors in our [`FuelLabs/fuel-connectors`](https://github.com/FuelLabs/fuel-connectors). Below you will find a snippet of using the current default connectors that are provided by the `fuel-connectors` package.
+
+<<< @/../../docs-snippets/src/guide/wallets/connectors.test.ts#fuel-options-connectors{ts:line-numbers}
+
+#### `storage`
+
+The `storage` is used internally to store the current connector state. This can be overriden as follows, by passing an instance that extends the `StorageAbstract` class.
```

Comment:
```suggestion
The `storage` is used internally to store the current connector state. It can be overridden by passing an instance that extends the `StorageAbstract` class.
```

Reply:
09e0871ef593c3cdbf7aa799b437996ef2232e90